pulsar-client/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -85,6 +86,15 @@
      */
     ConsumerBuilder topic(String topicName);
```

Comment:
Can we just change this to `topic(String... topicNames)` without adding `topics` method?

in `subscribeAsync()`, you can decide whether to subscribe to one topic or multiple topics.

internally in the builder you can maintain a `Set<String>` topics. so this allows you write in a more fluent style.

- subscribe to a topic: `builder.topic("topic-1")`
- subscribe to multiple topics: `builder.topic("topic-1", "topic-2", "topic-3")`
- subscribe to multiple topics in a chain way: `builder.topic("topic-1").topic("topic-2").topic("topic-3")`

in `subscribeAsync`: you can check `if (topics.size == 1)` to design what to do.

does this sound better? @merlimat

Reply:
It is good. @merlimat, Do you prefer this one?
If yes, will do the change.